code/__DEFINES/jobs.dm

```diff
@@ -109,69 +144,116 @@
 #define DEPT_BITFLAG_MED (1<<6)
 #define DEPT_BITFLAG_SEC (1<<7)
 #define DEPT_BITFLAG_VIP (1<<8)
-#define DEPT_BITFLAG_SILICON  (1<<9)
-// should check the ones in `\_DEFINES\economy.dm`
-// It's true that bitflags shouldn't be separated in two DEFINES if these are same, but just in case the system can be devided, it's remained separated.
+#define DEPT_BITFLAG_SILICON  (1<<9) // currently not used, but remained here just in case someone wants it
 
 //-------------------------------------------------------------------------------------------
 //------------------------------------- Job names -------------------------------------------
 //-------------------------------------------------------------------------------------------
+// ****** WARNING:
+// 		DO NOT CHANGE KEY. You are fine to change NAME defines, but DO NOT DO to KEY defines
+#define JOB_UNASSIGNED "Unassigned"
+#define JOB_DEMOTED "Demoted"
+
 // Command
 #define JOB_NAME_CAPTAIN "Captain"
+#define JOB_KEY_CAPTAIN "Captain"
 
 // Service
 #define JOB_NAME_HEADOFPERSONNEL "Head of Personnel"
+#define JOB_KEY_HEADOFPERSONNEL "Head of Personnel"
 #define JOB_NAME_ASSISTANT  "Assistant"
+#define JOB_KEY_ASSISTANT  "Assistant"
 #define JOB_NAME_BARTENDER  "Bartender"
+#define JOB_KEY_BARTENDER  "Bartender"
```

Comment:
Is this ever different? Why do we need duplicates of every single one of these defines?

Reply:
![image](https://user-images.githubusercontent.com/87972842/214168881-df99d995-5079-4080-9b19-5c397f225672.png)

changing name had some issue when we tried british PR before, and it can cause unexpectable errors in regard to job names, but differed key and name will make this much failproofed.